### DISCUSSION TITLE
Download large quantities of files with a select all button

### DISCUSSION THREAD

**Discussion by grimsby1hull0** [Tyrrrz/YoutubeDownloader#262] [General]
Would love to see this included! I hate thta I can only download baout 7 at a tme

**Comment by ScottsplaysSus** [comment]
I think I have a solution. Simply press ctrl and a together, and it should select all the files at once. Then, you can download them in one click. This works for me (Windows 10 21H1). Try it out and see if it works or not.

**Reply by grimsby1hull0** [reply]
Oh yeah, I've used that before for small amounts. I was trying to download 1249 videos overnight, just used another fork of Youtubedl called yt-dlp and it did the trick, just a shame i couldnt use this app for it, nice interface

**Reply by Tyrrrz** [reply]
You can press Ctrl+A too (first click select one video then do it). Typically standard Windows shortcuts/gestures should all work.

**Reply by grimsby1hull0** [reply]
But the issue is that it doesnt load the full amount of videos i want to download so I'm unable to select them all using CTRL+A because all the videos in the link arent being displayed, think the limit was 10
